docs/tuning-guide.md

```diff
@@ -210,6 +210,13 @@ rather than megabytes or smaller.
 Note that the GPU can encode Parquet and ORC data much faster than the CPU, so the costs of
 writing large files can be significantly lower.
 
+Use Hive Parquet or ORC tables instead of Hive Text table as the intermediate tables for
+CTAS(`Create Table As Select`) queries. The suggested change is to add `stored as parquet` for CTAS.
+Alternatively, you can set `hive.default.fileformat=Parquet` to create Parquet files by default.
+Refer to this
+[Hive Doc](https://cwiki.apache.org/confluence/display/Hive/Configuration+Properties#ConfigurationProperties-hive.default.fileformat)
+for more details.
```

Comment:
Sometimes users will disable spark.sql.hive.convertMetastoreOrc for some reason, shall we note that we need to turn it on to make sure orc hive table could run on GPU?

Reply:
sure added the notes about `spark.sql.hive.convertMetastoreOrc`